docs/migrating-configs.md

```diff
@@ -157,6 +157,99 @@ IP addresses and domain names can also include a port number (`1.2.3.4:80`).
   }
 }
 ```
+## From Version 3.1.0 to 3.2.0
+
+There are not any breaking changes between versions 3.1.0 and 3.2.0 of the configuration specification. Any valid 3.1.0 configuration can be updated to a 3.2.0 configuration by simply changing the version string in the config.
+
+The following is a list of notable new features, deprecations, and changes.
+
+### Partition Resizing
+
+The `partition` section gained a new `resize` field. When specified Ignition will resize an existing partition if it matches the config in all respects except the partition size. Refer to the [operator-notes][operator-notes] for more information.
+
+```json
+{
+  "ignition": { "version": "3.2.0" },
+  "storage": {
+    "disks": [{
+      "device": "/dev/sda",
+      "partitions": [{
+        "label": "root",
+        "sizeMiB": 16384,
+        "resize": "true"
+      }]
+    }]
+  }
+}
+```
+
+### LUKS
+
+Ignition now supports creating LUKS2 devices. Refer to the [operator-notes][operator-notes] for more information.
+
+```json
+{
+  "ignition": {"version": "3.2.0"},
+  "storage": {
+    "luks": [{
+      "name": "data",
+      "device": "/dev/sdb"
+    }],
+    "filesystems": [{
+      "path": "/var/lib/data",
+      "device": "/dev/disk/by-id/dm-name-data",
+      "format": "ext4",
+      "label": "DATA"
+    }]
+  },
+  "systemd": {
+    "units": [{
+      "name": "var-lib-data.mount",
+      "enabled": true,
+      "contents": "[Mount]\nWhat=/dev/disk/by-label/DATA\nWhere=/var/lib/data\nType=ext4\n\n[Install]\nWantedBy=local-fs.target"
+    }]
+  }
+}
+```
+
+### User/Group deletion
+
+The `passwd.users` & `passwd.groups` sections have a new field `shouldExist`. If false, Ignition will delete the specified user or group if it exists.
```

Comment:
I don't think we use the dotted notation anywhere else in this doc.  "The `passwd` `users` and `groups` sections"?

Let's say `If specified and false` to clarify that the default is true.